dataactbroker/fileRoutes.py

```diff
@@ -376,6 +399,11 @@ def certify_submission(submission):
         if submission.publish_status_id == PUBLISH_STATUS_DICT['published']:
             return JsonResponse.error(ValueError("Submission has already been certified"), StatusCode.CLIENT_ERROR)
 
+        window = gtas_window()
```

Comment:
If you're going to be calling this function outside the route, you should make it a helper function that is called from inside the route and here. You shouldn't be calling route functions from other functions generally. Maybe name it `get_gtas_window` and call it to get the gtas_window for each of these. It can just return `one_or_none` from the query.